techniques/system/common/1.0/update.st

```diff
@@ -269,32 +269,25 @@ bundle agent update_action
       "CfengineNovaExec"
         service_policy => "start",
         service_method => u_bootstart,
-        classes        => outcome("executor"),
+        classes        => success("executor_ok", "executor_kept", "executor_error"),
```

Comment:
These classes are the wrong order. It should be repaired, error, kept (see above)